store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10072,6 +10072,18 @@ TODO: delete them permanently from here
   <desc>Whether Video feature is enabled or not</desc>
 </attr>
 
+  <attr id="4094" name="zimbraFeatureMailRecallEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="domainInfo,accountInfo,domainInherited,accountCosDomainInherited" since="10.0.0">
+    <globalConfigValue>FALSE</globalConfigValue>
+    <defaultCOSValue>FALSE</defaultCOSValue>
+    <desc>Enables the mail recall functionality</desc>
+  </attr>
+
+  <attr id="4095" name="zimbraFeatureMailRecallTime" type="integer" min="1" max="30" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="domainInfo,accountInfo,domainInherited,accountCosDomainInherited" since="10.0.0">
+    <globalConfigValue>30</globalConfigValue>
+    <defaultCOSValue>30</defaultCOSValue>
+    <desc>Time(in minutes) within which a message can be recalled. The default time is 30 minutes. Only integer value is expected without any unit</desc>
```

Comment:
Please mention the valid/expected time limit also in description.

Reply:
yes, done.